helix-core/src/syntax.rs

```diff
@@ -73,16 +73,37 @@ pub struct IndentQuery {
     pub outdent: HashSet<String>,
 }
 
-fn read_query(language: &str, filename: &str) -> String {
-    static INHERITS_REGEX: Lazy<Regex> =
-        Lazy::new(|| Regex::new(r";+\s*inherits\s*:?\s*([a-z_,()]+)\s*").unwrap());
-
+#[cfg(not(feature = "embed_runtime"))]
+fn load_runtime_file(language: &str, filename: &str) -> Result<String, std::io::Error> {
     let root = crate::runtime_dir();
-    // let root = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    let path = root.join("queries").join(language).join(filename);
+    std::fs::read_to_string(&path)
+}
+
+#[cfg(feature = "embed_runtime")]
+use rust_embed::RustEmbed;
+
+#[cfg(feature = "embed_runtime")]
+#[derive(RustEmbed)]
```

Comment:
```suggestion
#[cfg(feature = "embed_runtime")]
#[derive(rust_embed::RustEmbed)]
```
Since this is just for a feature we can simplify this.